### ISSUE TITLE
[BUG] Switching to multi user mode creates file "users" instead directory "users"

### ISSUE THREAD

**Issue by BooneyNoobington** [Tzahi12345/YoutubeDL-Material#437]
Platform: Debian 11, AMD64

Installation:
- Installed requirements.
- Downloaded zip and decompressed.
- npm install
- npm audit fix
- npm start

Expected behaviour:
Switching to multi user mode in web ui should make it possible to download to directory ./users/.

Actual behaviour:
A file "users" is created. When downloading a video the following error is thrown:
```
2021-09-02T20:32:09.706Z INFO: Beginning migration: 4.1->4.2+
2021-09-02T20:32:09.718Z INFO: 4.1->4.2+ migration complete!
2021-09-02T20:32:11.149Z INFO: youtube-dl successfully updated!
2021-09-02T20:32:11.161Z INFO: YoutubeDL-Material v4.2 started on PORT 17442
(node:93259) UnhandledPromiseRejectionWarning: Error: ENOTDIR: not a directory, mkdir '/mnt/aStorage/Download/youtubedl-material/users/grindel/video'
    at Object.mkdirSync (fs.js:921:3)
    at Object.module.exports.makeDirSync (/mnt/aStorage/Download/youtubedl-material/node_modules/fs-extra/lib/mkdirs/make-dir.js:102:15)
    at /mnt/aStorage/Download/youtubedl-material/app.js:1096:16
    at new Promise (<anonymous>)
    at downloadFileByURL_exec (/mnt/aStorage/Download/youtubedl-material/app.js:1082:12)
    at /mnt/aStorage/Download/youtubedl-material/app.js:1920:28
    at Layer.handle [as handle_request] (/mnt/aStorage/Download/youtubedl-material/node_modules/express/lib/router/layer.js:95:5)
    at next (/mnt/aStorage/Download/youtubedl-material/node_modules/express/lib/router/route.js:137:13)
    at complete (/mnt/aStorage/Download/youtubedl-material/node_modules/passport/lib/middleware/authenticate.js:268:13)
    at /mnt/aStorage/Download/youtubedl-material/node_modules/passport/lib/middleware/authenticate.js:275:15
    at pass (/mnt/aStorage/Download/youtubedl-material/node_modules/passport/lib/authenticator.js:431:14)
    at Authenticator.transformAuthInfo (/mnt/aStorage/Download/youtubedl-material/node_modules/passport/lib/authenticator.js:453:5)
    at /mnt/aStorage/Download/youtubedl-material/node_modules/passport/lib/middleware/authenticate.js:272:22
    at IncomingMessage.req.login.req.logIn (/mnt/aStorage/Download/youtubedl-material/node_modules/passport/lib/http/request.js:55:13)
    at JwtStrategy.strategy.success (/mnt/aStorage/Download/youtubedl-material/node_modules/passport/lib/middleware/authenticate.js:253:13)
    at verified (/mnt/aStorage/Download/youtubedl-material/node_modules/passport-jwt/lib/strategy.js:115:41)

```

**Comment by GlassedSilver** [comment]
I'm using single user mode myself and that within a Docker environment, so I can't really help you here for sure, but @Tzahi12345 can chime in. :)

I could try spotting the code responsible for it and fixing it, but I don't wanna rush things and make it worse.

**Comment by Tzahi12345** [comment]
Apologies for the delay! Originally I thought this was a code issue, but long story short the auto builds have been bugged for a hot minute, create a users file instead of a directory as you suggested. [See fix here.](https://github.com/Tzahi12345/YoutubeDL-Material/commit/f9b8e786559dd6d8f9333385a54783ec9f24bff2)

I manually fixed the zip in the v4.2 release and pushed the fix, so this shouldn't come up again. In your case you can just create the folder yourself and it should work just fine. Let me know if you need any more help!
